metadata-ingestion/src/datahub/ingestion/source/metabase.py

```diff
@@ -232,7 +249,7 @@ def construct_dashboard_from_api_data(
             return None
 
         dashboard_urn = builder.make_dashboard_urn(
-            self.platform, dashboard_details.get("id", "")
+            self.platform, f"dashboard-{dashboard_details.get('id', '')}"
```

Comment:
we use urns as unique identifiers that are supposed to be stable over time, so changing the format from "{id}" to "dashboard-{id}" would be a breaking change

can we avoid a breaking change, or is there a reason it must be a breaking change?

Reply:
You are totally right! I misunderstood the code and thought the dashboard and the card id would conflict URNs but that's not true.
- https://github.com/datahub-project/datahub/blob/943bb57cbcf22db12c092a7f9a30c762aa2bf6e5/metadata-ingestion/src/datahub/ingestion/source/metabase.py#L256
- https://github.com/datahub-project/datahub/blob/943bb57cbcf22db12c092a7f9a30c762aa2bf6e5/metadata-ingestion/src/datahub/ingestion/source/metabase.py#L234